apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/steps/styled.ts

```
@@ -0,0 +1,580 @@
+  border-radius: var(--size);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  right: 0;
+  left: 0;
+  margin: auto;
+  animation: ${({ position }) => (position === 'left' ? 'appear-left' : position === 'right' ? 'appear-right' : 'none')}
+    2.5s cubic-bezier(0.19, 1, 0.22, 1) forwards;
+  animation-delay: ${({ position }) => (position === 'center' ? '0s' : '0.75s')};
+  border: ${({ position }) => (position === 'right' || 'center' ? `8px solid ${BLUE_COLOR}` : '0')};
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)